src/components/DropList/DropList.Combobox.jsx

```diff
@@ -67,11 +76,24 @@ function Combobox({
     },
 
     onInputValueChange({ inputValue }) {
-      setInputItems(
-        items.filter(item =>
-          itemToString(item).toLowerCase().startsWith(inputValue.toLowerCase())
-        )
+      let filtered = items.filter(item =>
+        itemToString(item).toLowerCase().startsWith(inputValue.toLowerCase())
       )
+      const isListEmpty = filtered.length === 0
+
+      if (isListEmpty && Array.isArray(customEmptyListItems)) {
+        const processed = customEmptyListItems.map(item => {
+          if (isFunction(item.customizeLabel)) {
+            item.label = item.customizeLabel(inputValue)
+            item.inputValue = inputValue
+          }
+
+          return item
+        })
+        filtered = filtered.concat(processed)
```

Comment:
Do we need to make concat here? It seems to be calling this code only if `isListEmpty`, so we might as well just replace the list, to not have to think here why there's `concat` :smile:

Reply:
Ahh yeah, good catch!